WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -737,6 +741,10 @@ function (ev) {
     var arr = this.widget.getSelection();
     if(arr && arr.length) {
         arr.sort();
+        // When retrieving data from view make sure to html decode it
+        arr = arr.map(function(val) {
```

Comment:
Created utility function in https://github.com/Zimbra/zm-ajax/pull/18
Used this utility function everywhere